framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -169,7 +170,21 @@ public static ConnectionPool getConnectionPool() {
         boolean foundTests = false;
         for (String testGroup : testGroups) {
           if (categories != null && !categories.contains(testGroup)) {
-            continue;
+            if (categories.contains("smoke") &&
```

Comment:
Counted 321 test definition files.

droptable suite uses droptable (2 json files)
maprdr suite uses functional_maprdb (1 json file)
some aggregate tests use "functional,smoke" (4 json files, including 2 under limit0)
impersonation suite uses impersonation (15 json files)
some suites use negative (10 json files)
complex uses regression (9 json files)
the rest use smoke (280 json files)